src/components/delegateRegistration/status/status.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import TransactionResult from '../../transactionResult';
+import DelegateAnimation from '../animations/delegateAnimation';
+import styles from './status.css';
+
+class Status extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      status: 'pending',
+    };
+
+    this.onCheckTransactionStatus = this.onCheckTransactionStatus.bind(this);
+    this.onRetry = this.onRetry.bind(this);
+  }
+
+  // istanbul ignore next
+  onCheckTransactionStatus() {
+    const { status } = this.state;
+    const { delegate } = this.props;
+
+    if (delegate.registerStep && delegate.registerStep === 'register-success' && status === 'pending') {
+      this.setState({ status: 'ok' });
+    }
+    if (delegate.registerStep && delegate.registerStep === 'register-failure' && status === 'pending') {
+      this.setState({ status: 'fail' });
+    }
+  }
+
+  // istanbul ignore next
+  onRetry() {
+    const { submitDelegateRegistration, userInfo } = this.props;
+    this.setState({ status: 'pending' });
+    submitDelegateRegistration(userInfo);
+  }
+
+  render() {
+    const { t, goBackToDelegates } = this.props;
+    const { status } = this.state;
+
+    const isTransactionSuccess = status !== 'fail';
+    // istanbul ignore next
```

Comment:
Can be removed after resolving my comment in `src/components/delegateRegistration/delegateeRegistration.test.js`

Reply:
yes this is going to be addressed in PR https://github.com/LiskHQ/lisk-hub/pull/2186